js/plugins/google-cloud/tests/traces_test.ts

```diff
@@ -14,10 +14,13 @@
  * limitations under the License.
  */
 
+import { runWithRegistry } from '@genkit-ai/core/registry';
 import { ReadableSpan } from '@opentelemetry/sdk-trace-base';
 import { Genkit, genkit, run, z } from 'genkit';
 import assert from 'node:assert';
 import { after, before, beforeEach, describe, it } from 'node:test';
+import { generate } from '../../../ai/src/generate.js';
```

Comment:
Should these be different imports?

Reply:
Yes, they should probably be `from '@genkit-ai/ai`

Reply:
it actually looks like it's now 'genkit' and 'genkit/model'